newdle/client/src/components/answer/AnswerPage.js

```diff
@@ -88,11 +88,14 @@ export default function AnswerPage() {
   const participantAnswers = useSelector(getParticipantAnswers);
   const participantHasAnswers = !!Object.keys(participantAnswers).length;
   const participant = useSelector(getParticipant);
+  const [_comment, setComment] = useState(undefined);
+  const comment = _comment == null && participant ? participant.comment : _comment;
```

Comment:
why not use `null` for the default state? then you can use `===` here instead of using the null/undefined coercion of `==`

Reply:
I had null initially. But I wanted to avoid the unpredictable case of having the input `value={null}`. 
It will complain something like: "set as undefined instead of null..."

But if you agree that this is not worth the concern, I can change it.

Reply:
won't this be covered by `comment || ''` in the input?

Reply:
Yes, it will be. I added that shortly after and eventually forgot it was even there.